.gitignore

```diff
@@ -26,7 +26,6 @@ build.ninja
 *.z64
 *.Yay0
 *.msg.h
-*.bin
```

Comment:
This is to enable people to override `.bin` files with an asset stack. We already ignore the dump asset directories